### ISSUE TITLE
`popover` content not read back to screen reader users on demand

### ISSUE THREAD

**Issue by geospatialem** [Esri/calcite-components#4865]
### Summary

When accessed via a reference element `popover` content is not read back to screen reader users on demand.

### Actual Behavior

The `popover` content is not read back to screen reader users when accessed via the reference element. 

*However, as a workaround, users can press the down arrow to read back the content, but this is not expected behavior for screen reader users.*

### Expected Behavior

Upon engaging the `popover` from the reference element, content should be provided politely, perhaps using the `aria-role='status'`, to provide context to screen reader users.

Similar to the current functionality, after completion, the focus should return to the reference element.

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/popover/#sample

### Reproduction Steps

1. Enable a screen reader (NVDA, JAWS or VoiceOver)
2. Navigate to the [sample](https://developers.arcgis.com/calcite-design-system/components/popover/#sample)
3. With the screen reader active, navigate via keyboard to focus on the "Activate popover" button to trigger the popover
4. Observe no context is provided to the screen reader initially.
    - However, if using the down arrow key, context will be read back to the user

### Reproduction Version

beta.83

### Relevant Info

- Windows (JAWS and NVDA)
- Mac OS (Safari/VoiceOver)
- related to: #4864

**Comment by driskull** [comment]
@geospatialem popover already has a role of `role="dialog"`. Do you know what should be added exactly?

**Comment by geospatialem** [comment]
> @geospatialem popover already has a role of `role="dialog"`. Do you know what should be added exactly?

@driskull thanks for taking a peek at this issue! This will be the same resolution as the `tooltip` screen reader fix in adding an `aria-live="polite"` for JAWS, NVDA, and VoiceOver.

**Comment by benelan** [comment]
verified on master
